test/gtest/uct/ib/test_cqe_zipping.cc

```diff
@@ -63,80 +46,77 @@ struct test_cqe_zipping : public uct_test {
 
     size_t collect_zipped_counters()
     {
-        size_t zipped_cqes_cnt =
-                UCS_STATS_GET_COUNTER(iface_stats(*peers[0]),
-                                      UCT_IB_IFACE_STAT_RX_COMPLETION_ZIPPED) +
-                UCS_STATS_GET_COUNTER(iface_stats(*peers[0]),
-                                      UCT_IB_IFACE_STAT_TX_COMPLETION_ZIPPED) +
-                UCS_STATS_GET_COUNTER(iface_stats(*peers[1]),
-                                      UCT_IB_IFACE_STAT_RX_COMPLETION_ZIPPED) +
-                UCS_STATS_GET_COUNTER(iface_stats(*peers[1]),
-                                      UCT_IB_IFACE_STAT_TX_COMPLETION_ZIPPED);
-        return zipped_cqes_cnt;
-    }
-
-    void do_test()
-    {
-        ucs_time_t deadline = ucs::get_deadline(60);
-        /* Some transports (e.g. UD) dynamically extend the amout of sends,
-         * so we must do several send/recv iterations to make CQE zipping
-         * sensable from the FW side.
-         */
-        while (ucs_get_time() < deadline) {
-            send_cnt = recv_cnt = 0;
-            flush();
-            send_while_possible();
-            wait_for_completion();
-
-            if (collect_zipped_counters() > 0) {
-                return;
+        std::vector<int> counters = {UCT_IB_IFACE_STAT_RX_COMPLETION_ZIPPED,
+                                     UCT_IB_IFACE_STAT_RX_COMPLETION_ZIPPED};
+        size_t zipped_cqes_cnt{};
```

Comment:
`{}` ?

Reply:
Didn't understand the question. Just a default initialization, what is the problem?

Reply:
`{}` is usually used for classes/structs, etc.
Why not `= 0`?

Reply:
> {} is usually used for classes/structs, etc.

Why do you think so?
There is no special reason, I just used one of possible ways to init the variable. I can change it if you like.

Reply:
You are right that it is possible, but AFAIK initializer lists are rarely used for vars. I can't find even a single example in our repo, so I'd say to avoid it even just for the sake of consistency :)

Reply:
```suggestion
        size_t zipped_cqes_cnt = 0;
```